src/components/parameters-tabs.tsx

```diff
@@ -258,7 +247,7 @@ const ParametersTabs: FunctionComponent<OwnProps> = (props) => {
         });
     }, [enableDeveloperMode]);
 
-    const loadFlowStatus = useSelector((state: AppState) => state.computingStatus[ComputingType.LOAD_FLOW]);
+    const computationStatus = useSelector((state: AppState) => state.computingStatus[tabValue as ComputingType]);
```

Comment:
for SHORT_CIRCUIT_ONE_BUS computation, this does not work, because here, we have 2 distinct computation type (SHORT_CIRCUIT and SHORT_CIRCUIT_ONE_BUS) for just one parameter tab value (TAB_VALUES.shortCircuitParamsTabValue)